doc/build.md

```diff
@@ -25,7 +25,7 @@ e.g `./helloworld_1.0.0_amd64.snap`
 ### Create Snap
 
 Start task `Snapcraft` by selecting the following command of the menu bar in Visual Studio Code Editor:  
-__Terminal > Run Build Task__  or press __Ctrl-P__ and enter __task CMake Build Snap__. Select the target for which you want to create the snap.
+__Terminal > Run Build Task__  or press __Ctrl-P__ and enter __task Snapcraft__. Select the target for which you want to create the snap.

```

Comment:
We have to take care, CMake based projects have the CMake build task.

Maybe a solution "CMake Build Snap (for CMake Projects) or Snapcraft for the other projects" would be better